storage/netapp/netapp_e_hostgroup.py

```diff
@@ -0,0 +1,426 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, NetApp, Inc
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+DOCUMENTATION = '''
+module: netapp_e_hostgroup
+version_added: "2.2"
+short_description: Manage NetApp Storage Array Host Groups
+author: Kevin Hulquest (@hulquest)
+description:
+- Create, update or destroy host groups on a NetApp E-Series storage array.
+options:
+  api_username:
+      required: true
+      description:
+      - The username to authenticate with the SANtricity WebServices Proxy or embedded REST API.
+  api_password:
+      required: true
+      description:
+      - The password to authenticate with the SANtricity WebServices Proxy or embedded REST API.
+  api_url:
+      required: true
+      description:
+      - The url to the SANtricity WebServices Proxy or embedded REST API.
+      example:
+      - https://prod-1.wahoo.acme.com/devmgr/v2
+  validate_certs:
+      required: false
+      default: true
+      description:
+      - Should https certificates be validated?
+  ssid:
+    required: true
+    description:
+    - The ID of the array to manage (as configured on the web services proxy).
+  state:
+    required: true
+    description:
+    - Whether the specified host group should exist or not.
+    choices: ['present', 'absent']
+  name:
+    required: false
+    description:
+    - The name of the host group to manage. Either this or C(id_num) must be supplied.
+  new_name:
+    required: false
+    description:
+    - specify this when you need to update the name of a host group
+  id:
+    required: false
+    description:
+    - The id number of the host group to manage. Either this or C(name) must be supplied.
+  hosts::
+    required: false
+    description:
+    - a list of host names/labels to add to the group
+  api_username:
+    required: true
+    description:
+    - the user name for authentication into the web services proxy
+  api_password:
+    required: true
+    description:
+    - the password for authentication into the web services proxy
+  api_url:
+    required: true
+    description:
+    - the resolvable endpoint for the web services proxy. Example, http://10.10.10.10:8080/devmgr/v2
```

Comment:
Fixed.  I think ansible-validate-modules didn't catch this because the documentation didn't have the --- yaml declaration?  Not sure on this but it would be interesting to know.